### PR TITLE
[QC/RB/Stats] add missing fields from MRI PARAMETER FORM

### PR DESCRIPTION
**Pull request by ridz1208**
### Brief summary of changes
- Add quality control doespageload test
- add missing fields for mri_parameter_form

To test this:
- check the QC module works
- check the stats module loads the all tables

Note some files from `RB_files/` in the diff seem to contain changes in ordering of fields. the only important file with actual changes is `RB_files/RB_mri_parameter_form.sql`

remake of https://github.com/aces/Loris/pull/4800

### CODE REVIEW

File: raisinbread/instruments/instrument_sql/mri_parameter_form.sql [hunk 1]
```diff
@@ -26,6 +26,27 @@ CREATE TABLE `mri_parameter_form` (
 `2d_flair_Scan_Done` enum('yes','partial','no') default NULL,
 `2d_flair_number_attempts` varchar(255) default NULL,
 `2d_flair_comments` text default NULL,
+`fieldmapBOLD_Scan_Done` enum('yes','partial','no') DEFAULT NULL,
```
Comment (driusan) [inline raisinbread/instruments/instrument_sql/mri_parameter_form.sql]: Did you verify that these match:
1. the mri_scan_types table
2. the instrument itself used by raisinbread?

Comment (ridz1208) [reply raisinbread/instruments/instrument_sql/mri_parameter_form.sql]: 1. yes but whoever is codereviewing should confirm that
2. no these fields will not show in the instrument. I am doing this simply to bugfix modules that are too dump to handle a missing field.

future changes should include. 
- adding these fields to the instrument front end
- improving quality control to be more elegant in handling the missing columns

File: modules/quality_control/test/quality_controlTest.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    }
+    /**
+     * Delete test data
+     *
+     * @return void
+     */
+    function tearDown()
+    {
+        parent::tearDown();
+    }
```
Comment (driusan) [inline modules/quality_control/test/quality_controlTest.php]: overriding the parent just to explicitly call the parent seems unnecessary

Comment (ridz1208) [reply modules/quality_control/test/quality_controlTest.php]: Hahaha i just copied from the other PR.

Can someone push to my branch ?

Comment (kongtiaowang) [reply modules/quality_control/test/quality_controlTest.php]: @ridz1208 I push the change.

File: modules/quality_control/test/quality_controlTest.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ * @category Test
+ * @package  Loris
+ * @author   Wang Shen <wangshen.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+ require_once __DIR__
+    . "/../../../test/integrationtests/LorisIntegrationTest.class.inc";
+/**
+ * AcknowledgementsIntegrationTest
```
Comment (driusan) [inline modules/quality_control/test/quality_controlTest.php]: This doesn't match the class name (QualityControl vs Acknowledgements), same with the first line of this file.

**Comment by kchatpar** [comment]
I'm not able to load the QC module @ridz1208 after clearing my database and resourcing the raisinbread dataset. The issue of requiring the file RB_mri_parameter_form_for_QC.sql in #4800 is still present. 

I'm getting the error: PHP Fatal error:  Uncaught DatabaseException: Unknown column 'm.T1_Scan_done' in 'field list' in /var/www/loris/php/libraries/Database.class

**Comment by kongtiaowang** [comment]
@kchatpar 
in mri_parameter_form.sql, there is "T1_Scan_done" column, could you check your database to make sure your table has this column. If it still doesn't work, please remove the "pass manual test" tag.

**Comment by kchatpar** [comment]
I was able to get the QC module to load. LGTM!

**Pull request merged**